### PR TITLE
add rfc as a pending state

### PR DESCRIPTION
**Pull request by laijs**
Signed-off-by: Lai Jiangshan <jiangshanlai@gmail.com>

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ always_pending:
   labels:
     - do-not-merge
     - wip
+    - rfc
```
Comment (jodh-intel) [inline .pullapprove.yml]: How about expanding `title_regex` for this too. Something like:

```
title_regex: '(WIP|RFC)'
```

**Comment by laijs** [comment]
@jodh-intel updated, thanks!

**Comment by jodh-intel** [comment]
Hi @laijs - sorry, I didn't mean that you needed to remove `rfc` from the list of labels. I just meant that you could *also* add the `RFC` to `title_regex` so that the PR will be blocked if it has *either* the `rfc` label, *or* "`RFC`" in its title.

**Comment by laijs** [comment]
@jodh-intel updated again! thanks!

File: .pullapprove.yml [hunk 2]
```diff
@@ -6,10 +6,11 @@ requirements:
 
 # Disallow approval of PRs still under development
 always_pending:
-  title_regex: 'WIP'
+  title_regex: 'WIP|RFC'
```
Comment (sameo) [inline .pullapprove.yml]: @laijs I think you want parenthesis around that: `(WIP|RFC)`

Comment (laijs) [reply .pullapprove.yml]: o, I should copy what @jodh-intel has said.

Comment (laijs) [reply .pullapprove.yml]: @sameo updated, thanks,

**Comment by jodh-intel** [comment]
lgtm

**Comment by sameo** [comment]
LGTM

**Pull request merged**